.github/workflows/status-checks.yml

```diff
@@ -5,7 +5,6 @@ on:
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master ]
```

Comment:
Originally this was an PR on top of another PR (`storage-encryption`). Due to that line the status checks were not run on that PR. I wasn't sure whether all unit tests passed, because my local setup was a bit buggy at at time, so I wanted GitHub to validate that for me. Therefore, I deleted the requirement that a PR should target `master`. In my view it is nice to keep this and always run a check on every PR.